supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala

```diff
@@ -268,22 +266,24 @@ class DispatchedConfigAsyncJobExecutionActor(override val standardParams: Standa
         SharedFileSystemRunStatus("Done")
       case Some(s) if s.status == "Running" =>
         // Exitcode file does not exist at this point, checking is jobs is still alive
-        if (exitCodeTimeout.isEmpty) s
-        else if (isAlive(handle.pendingJob).fold({ e =>
-          log.error(e, s"Running '${checkAliveArgs(handle.pendingJob).argv.mkString(" ")}' did fail")
-          true
-        }, x => x)) s
-        else SharedFileSystemRunStatus("WaitingForReturnCode")
+        exitCodeTimeout match {
+          case Some(timeout) =>
+            if (s.experired(timeout)) s
```

Comment:
I'm not sure I understand the logic, this seems to say "if we're past the timeout, do nothing, otherwise check if the job is still alive". Isn't that the opposite of the intent ?